openerp/addons/base/res/res_partner.py

```diff
@@ -87,6 +70,9 @@ def fields_view_get_address(self, cr, uid, arch, context={}):
                     tree = etree.fromstring(v)
                     # needs to apply modifiers
                     for childnode in tree.getchildren():
+                        if 'use_parent_address' in self._all_columns:
```

Comment:
But this is going to apply the modifier to each element found, including div and so on, doesn't it?